osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/dlmsobjectconfig/DlmsObjectConfigSmr50.java

```diff
@@ -55,7 +55,7 @@
 
     @Override
     List<Protocol> initProtocols() {
-        return Arrays.asList(Protocol.SMR_5_0, Protocol.SMR_5_1);
+        return Arrays.asList(Protocol.SMR_5_0_0, Protocol.SMR_5_1);
```

Comment:
SMR_5_1 ... was al zo, dus zal wel kloppen

Reply:
ja idd, het is dus 5.0.0 (komt uit de shipment file) en 5.1